content/en/logs/explorer/advanced_filtering.md

```diff
@@ -0,0 +1,89 @@
+    - The results from the **NOT IN** operator exclude logs where the attribute value is found in the subquery results. For example, if you only want to see `status:error` logs, but also want to filter out `status:error` logs where users associated with those logs eventually end up with a `status:success` log. See [Filter outdated or superseded logs](#filter-outdated-or-superseded-logs) for a detailed example.  
+1. Optionally, reduce the number of subquery attribute values to match against. The default value and maximum value is `1000`. Choose between **top** (highest frequency values) or **bottom** (lowest frequency values).
+
+### Subquery examples
+
+The following are scenarios where you need to use subqueries to get the information you need from your logs.
+
+#### Filter outdated or superseded logs
+
+Assume you operate an e-commerce platform. A log is generated each time one of your customers tries to place an order. You want to analyze your logs to understand the total value of potential purchases lost due to an ongoing issue with your website.
+
+You realize, however, that an order can fail multiple times before successfully completing: meaning for that particular order ID, there are log entries in the search results for both `status:error` logs and `status:success` logs. If you extracted lists of unique order IDs from the two queries, this order ID would appear in both. With subqueries, you can obtain a mutually exclusive list.
+
+In this example, you are only interested in logs for orders that did not eventually succeed. To filter out orders that eventually succeeded using the subquery feature:
+
+1. Define a subquery for `status:success` logs.
+1. Select the **NOT IN** operator to exclude orders from the subquery result set.
+
+{{< img src="logs/explorer/advanced_filtering/filter_outdated_example.png" alt="The query editor showing the setup for filtering out orders that ended up succeeding" style="width:100%;" >}}
+
+#### Correlate across different log sources
+
+Assume you have a service named `network_directory` that monitors all internal network resources and access to those resources within your organization. Log events generated by this service include standard attributes (like `host`, `service`, `source`) and custom attributes like the client's IP address.
+
+You have another `device-manager` service that tracks all internal assets (infrastructure, employee devices, and so on.)
+
+You are investigating an ongoing attack and there is a significant increase in API requests across almost all of your endpoints. You want to first identify IP addresses associated with anomalous request volumes so that you can quickly block them at the firewall level. However, your internal services are some of the largest consumers of these endpoints and you need to exclude them from your query results to avoid mistakenly blocking them.
```

Comment:
```suggestion
You are investigating an ongoing attack and observe there is a significant increase in API requests across almost all of your endpoints. You want to first identify IP addresses associated with anomalous request volumes so that you can block them at the firewall level. However, your internal services are some of the largest consumers of these endpoints and you need to exclude them from your query results to avoid mistakenly blocking them.
```